### ISSUE TITLE
Remove turn off/reboot menu

### ISSUE THREAD

**Issue by efeme4** [XorTroll/Goldleaf#706]
## Feature description

The title explains it all, I don't think a file manager should have a direct turn off button with no confirmation more than pressing A. I've turned off my switch on accident several times thanks to this awesome feature just because I had my controller on my  lap and I pressed zr on accident while trying to access a folder. The switch already has an entire button and integrated menus entirely dedicated to that, so I think goldleaf doesn't need such a thing.

## Benefits

Being able to keep your console turned on when you want and avoid unwanted shutdowns or reboots.
Goldleaf would be way more comfortable to use.

**Comment by XorTroll** [comment]
I'd be willing to remove it or more likely make it opt-in if this is a common thing, but I'd say I never heard similar complaints
At least for me as a developer, having this instead of the usual power button holding is really convenient

**Comment by Masamune3210** [comment]
Instead of removing it, just maybe add a "are you sure?". having a way to turn off the system is quite useful

**Comment by efeme4** [comment]
Yes, maybe optionality instead of removal is the solution, but I can assure you that the current way of working leads to accidental shutdowns.

**Comment by efeme4** [comment]
If that's true, then I think we can finally close this issue. Thanks a lot for addressing this problem.
